tests/test_examples.py

```diff
@@ -0,0 +1,6 @@
+from btrack import datasets
+
+
+def test_pooch_registry():
+    """Test that `pooch` registry is up to date with remote version."""
```

Comment:
```suggestion
    """
    Test that `pooch` registry is up to date with remote version.
    This will fail if the remote file does not match the hash hard-coded
    in btrack.datasets.
    """
```